### PR TITLE
GEOMESA-876 Move csv package from accumulo-ds

### PR DESCRIPTION
**Pull request by aheyne**
Moved the csv package from accumulo-datastore to geomesa-utils.

Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

**Comment by aheyne** [comment]
This applies to GEOMESA-1352 and GEOMESA-876

File: geomesa-accumulo/geomesa-accumulo-datastore/pom.xml [hunk 1]
```diff
@@ -97,10 +93,12 @@
             <groupId>org.calrissian.mango</groupId>
             <artifactId>mango-core</artifactId>
         </dependency>
+        <!--
```
Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-datastore/pom.xml]: can delete this commented out import

File: geomesa-utils/pom.xml [hunk 2]
```diff
@@ -84,6 +88,10 @@
             <artifactId>gt-data</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-shapefile</artifactId>
+        </dependency>
+        <dependency>
```
Comment (elahrvivaz) [inline geomesa-utils/pom.xml]: is this needed?

Comment (aheyne) [inline geomesa-utils/pom.xml]: Yeah, the csv package uses ShapefileDataStore and ShapefileDataStoreFactory.

**Comment by aheyne** [comment]
Merged master and removed commented out dependency in pom.

File: geomesa-utils/pom.xml (hunk 2 above)
Comment (jnh5y) [inline geomesa-utils/pom.xml]: *blinks* I wonder if we could use a geomesa-shapefile module or something to line up dependencies, etc.

**Pull request merged**